<launch>
    <node pkg="gstreamer_ros_bridge" type="gstreamer_bridge_node" name="gstreamer_bridge_node" output="screen">
        <!--params for gstreamer pipeline to peer (over udp)-->
        <param name="bitrate" value="1200"/>
        <param name="mtu" value="500"/>
        <param name="gst_width" value="160"/>
        <param name="gst_height" value="90"/>
        <param name="gst_fps" value="15"/>
        <param name="gst_topic" value="/camera/image_rect"/>

        <!--target ip and port-->
        <param name="ip" value="10.7.0.2"/>
        <param name="port" value="5602"/>
    </node>
</launch>
